lib/galaxy/managers/configuration.py

```diff
@@ -56,7 +56,7 @@ def version(self) -> Dict[str, Any]:
             with open(json_file) as f:
                 extra_info = json.load(f)
         except OSError:
-            log.info('Galaxy JSON version file not loaded')
+            log.warning('Galaxy JSON version file %s not loaded.' % json_file)
```

Comment:
```suggestion
            log.warning('Galaxy JSON version file %s not loaded.', json_file)
```
Slightly more efficient (even w.r.t to an f-string), because the interpolation is postponed after checking the log level.

Not sure I agree with the log level change, since the JSON version is not shipped by default.